src/asmcnc/comms/logging/log_exporter.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+import os
+from asmcnc.production.database import credentials as creds
+import paramiko
+
+WORKING_DIR = 'C:\\SBLogs\\'
+
+export_logs_folder = '/home/pi/exported_logs'
+
+
+def create_log_folder():
+    if not os.path.exists(export_logs_folder) or not os.path.isdir(export_logs_folder):
+        os.mkdir(export_logs_folder)
+
+
+def generate_logs():
+    create_log_folder()
+
+    str_current_time = datetime.now().strftime("%m-%d-%Y-%H-%M-%S")
+
+    log_name = "easycut-exported-logs-" + str_current_time + ".txt"
+    full_path = export_logs_folder + '/' + log_name
+
+    command = "journalctl > " + full_path
+
+    os.system(command)
+
+    return log_name
+
+
+def trim_logs(log_file_path, x_lines):
```

Comment:
purpose of this?

Reply:
me and Ed discussed, may be useful for future application to be able to trim down. e.g. if we used this system for automatic customer support logs, it would be easier for less familiar people like Elliot if there wasn't thousands of lines to go through